bgpd/bgp_route.c

```diff
@@ -9674,7 +9674,7 @@ void route_vty_out_tag(struct vty *vty, const struct prefix *p,
 
 	label = decode_label(&path->extra->label[0]);
 
-	if (bgp_is_valid_label(&label)) {
+	if (bgp_is_valid_label(&path->extra->label[0])) {
```

Comment:
Maybe we should do `decode_label()` after we validate it?

Reply:
Yes, I think that's more reasonable. But I'm not sure if we should print the original label or the results after `decode_label()` in this place.

Reply:
I think yes.